### ISSUE TITLE
Location do not contains fields

### ISSUE THREAD

**Issue by ibaca** [google/elemental2#2]
The closure externs seem ok, but elemental does not contain Location fields (`href`, `protocol`, `search`, etc).
https://github.com/google/closure-compiler/blob/master/externs/browser/ie_dom.js#L1252

I think this ie_dom is the one used because it is the unique place where 'forceReload' appears.

Current elemental2 Location class:
```java
@JsType(isNative = true, namespace = JsPackage.GLOBAL)
public interface Location {
  void assign(String url);
  void reload();
  void reload(boolean forceReload);
  void replace(String url);
}
```

**Comment by tbroyer** [comment]
`ie_dom.js` is not used: https://github.com/google/elemental2/blob/887b489c0911965f106a143a4c48a8da925bb9e1/java/elemental2/dom/BUILD

`Location` type comes from https://github.com/google/elemental2/blob/70c55ead394b07f64329b7ad0c9f8206e6d211a0/java/elemental2/dom/dom_missing_api.js#L25

**Comment by jDramaix** [comment]
Should be on `DomGlobal.window.location`

**Comment by tbroyer** [comment]
@jDramaix I think the issue is that `Location` only has methods `apply`, `reload` and `replace`, and none of the `href`, `protocol`, `search`, etc. properties: https://developer.mozilla.org/en-US/docs/Web/API/Location

**Comment by tbroyer** [comment]
Ping @jDramaix, I believe this needs to be reopened.

**Comment by gkdn** [comment]
@jDramaix dom_missing_api.js is missing plenty of Location APIs.

**Comment by gkdn** [comment]
@ibaca we are not using ie_dom.js; feel free send us a patch that moves more APIs from ie_dom.js but pls update the documentation similar to other ones in dom_missing_api.js. We will eventually fix them in the Closure and get rid off dom_missing_api.js.

**Comment by jDramaix** [comment]
Move the API in dom_missing_api.js if you are sure it's not a specific api of one browser. We'll clean the externs files at some point.

**Comment by jDramaix** [comment]
Fixed with #9
